_posts/2019/2019-03-26-behind-the-scenes-localization-at-epages.md

```diff
@@ -0,0 +1,96 @@
+
+### Hi Christina. Thank you for taking the time to do this interview. 
+
+I'm happy to.
+All of a sudden I had this appointment in my calendar 😉.
+
+### Could you tell us a bit about your background, and how you got into your role as a Localization Manager?
+
+Sure.
+I studied International Technical Communication (MA Internationale Fachkommunikation) at the University of Hildesheim, as I was interested in the combination of language and technology.
+A part of my studies was a mandatory internship which I did at ePages.
+During my internship, I was so fascinated by the company and the colleagues, that I decided to also write my master thesis here.
+
+It turned out that localization was a good topic, because ePages had already discovered that there were some things to improve.
+The internal processes between UI writers and developers, as well as the translation process required a closer look.
+And the scope was suitable for a master thesis.
+
+Now that I was so familiar with the topic of software localization, and introduced new processes as well as a localization tool, I stayed as a Localization Manager.
+
+### How does localization of ecommerce software work?
+
+It's not so much different from localizing any other software, I guess.
+But we work agile.
+We have many small features at short intervals, and therefore smaller translation packages.
+For this reason (and several others), we decided to use PhraseApp as our localization tool.
+
+### Why PhraseApp?
+
+It facilitated the communication with the developers A LOT.
+They can concentrate on development, while I can still arrange the coordination and translation of new features in a timely manner.
+What's more, PhraseApp comes with a Translation Memory and an In-Context Editor. 
+Especially the topic of providing context to our translators was a big issue before we introduced the tool.
+
+### How much programming must a Localization Manager know to work on software localization?
+
+Well, you don't need to be a developer, but it helps if you understand what they do and how they do it.
+For example, while optimizing our processes, we also improved how developers should craft translation keys.
+If developers do translation-related Pull Requests, I check the keys accordingly.
+And sure, it's beneficial to know the GitHub stuff.
+
+### How many languages do you take care of? Do you have to speak all of them fluently?
+
+Oh, it would be nice, if I could speak all of our 16 languages fluently!
+Internally we take care of German, English US, and English UK.
+But I'm not so much involved in the process of crafting UI texts.
+It's more about having the complete overview of the whole process, coordinating the texts with our translation agency, and being in touch with the developers, and UI writers, and such things.
+
+### What challenges have you faced so far?
+
+I'm with ePages since November 2017, and yes, there were some challenges 😉.
```

Comment:
When reading this again, it felt like I somehow "ignored" the initial question which was not "have there been challenges". So maybe it's better to start with something like this: 

Since I've started as a Localization Manager in November 2017, there were already various challenges 😉.